server/src/main/java/com/cloud/projects/ProjectManagerImpl.java

```diff
@@ -661,6 +661,11 @@ public void doInTransactionWithoutResult(TransactionStatus status) throws Resour
                     _projectDao.update(projectId, project);
                 }
 
+                if (name != null) {
+                    project.setName(name);
+                    _projectDao.update(projectId, project);
+                }
+
```

Comment:
Don't think we need to update twice using Dao when both name and display text are being updated